### PR TITLE
Added in ability to use TMTpro-18 and corrected chemical formula for hydroxybutyrylation

### PR DESCRIPTION
**Pull request by nbollis**
Added in the ability to use TMT18 and two unit tests to go with it. 
Noticed the chemical formula for hydroxybutylation was incorrect and this request fixes it.

### CODE REVIEW

**Comment by nbollis** [comment]
Removed extra label in TMT11 that was incorrect
Reordered the testcases to be in ascending mass order.

**Review by kyp4 — APPROVED** [review]
Looks good to me - all unit tests passed and found TMT18 in a test data set

File: EngineLayer/Mods/Mods.txt [hunk 1]
```diff
@@ -155,7 +155,7 @@ TG   K
 PP   Anywhere.
 MT   Common Biological
 DI   HCD:169.110323533
```
Comment (acesnik) [inline EngineLayer/Mods/Mods.txt:157]: Good catch on this. C4H7O2 looks right.

Will this change the `DI` mass? Should that be updated? I'm not sure how that's calculated.

File: EngineLayer/Mods/tmt.txt [hunk 2]
```diff
@@ -1,4 +1,89 @@
...
+#NL   HCD:C1 N1 H15 C{13}7
+DI   HCD:C1 N1 H15 C{13}7
+#NL   HCD:C1 H15 C{13}7 N{15}1
+DI   HCD:C1 H15 C{13}7 N{15}1
+#NL   HCD:N1 H15 C{13}8
+DI   HCD:N1 H15 C{13}8
+#NL   HCD:H15 C{13}8 N{15}1
+DI   HCD:H15 C{13}8 N{15}1
+//
+TMT Multiplex
```
Comment (acesnik) [inline EngineLayer/Mods/tmt.txt:86]: Should this line be deleted? Looks like the title was copy-pasted here

Comment (acesnik) [reply EngineLayer/Mods/tmt.txt:86]: I guess this is true below, too, so no worries

**Review by acesnik — APPROVED** [review]
This looks good. Nice work getting the mzML in there for testing actual data.

Just had one question below, but not sure it needs addressing.

**Review by trishorts — APPROVED** [review]
good work nic

**Review by rmmiller22 — APPROVED** [review]

**Pull request merged**